Resources/Storage/test/Test_StorageElement.py

```diff
@@ -81,6 +81,79 @@ def mock_StorageFactory_generateStorageObject( storageName, pluginName, paramete
 
   return S_OK( storageObj )
 
+def mock_StorageFactory_getConfigStorageName( storageName, referenceType, rootConfigPath ):
+  return S_OK( storageName )
+
+def mock_StorageFactory_getConfigStorageOptions( storageName, rootConfigPath, derivedStorageName = None ):
+  """ Get the options associated to the StorageElement as defined in the CS
+  """
+
+  options = {'BackendType': 'local',
+                            'ReadAccess': 'Active',
+                            'WriteAccess': 'Active',
+            }
+
+  if storageName in ( 'StorageE', ):
+    options['WriteProtocols'] = ['root', 'srm']
+
+
+
+
+  return S_OK( options )
+
+def mock_StorageFactory_getConfigStorageProtocols( storageName, rootConfigPath, derivedStorageName = None ):
```

Comment:
This is also in Test_FilePlugin.py but I start to understand that you may need different ones. 

In general, anyway these are not mock objects, they are more simply "fake objects". 
And one more thing: if there's a mock object that can be used by several test modules, we tend to put them in a single module, e.g. see files mock_FC.py and mock_DM.py.